nativelib/src/main/scala/scala/scalanative/native/stdlib.scala

```diff
@@ -57,4 +57,6 @@ object stdlib {
   def EXIT_FAILURE: CInt = extern
   @name("scalanative_libc_rand_max")
   def RAND_MAX: CInt = extern
+  @name("scalanative_libc_environ")
+  def environ: Ptr[CString] = extern
 }
```

Comment:
`environ` comes from `unistd.h`, not `stdlib.h`. Bindings for `unistd` should live in `scala.scalanative.posix.unistd` instead.